irods_specific_implementation/irods_rest_query_api_implementation.h

```diff
@@ -39,6 +39,12 @@ namespace irods::rest
                 auto conn = get_connection(_request.headers().getRaw("authorization").value(), _query_string);
 
                 std::string query_string{decode_url(_query_string)};
+                if ("0" == _case_sensitive) {
+                    std::transform(query_string.begin(),
+                                    query_string.end(),
+                                    query_string.begin(),
+                                    [](unsigned char c){ return std::toupper(c); });
```

Comment:
indentation is a little off

Reply:
And if we care to keep the formatting like this, need to surround the lines with the special comments to make clang-format happy:
```c++
// clang-format off
...
// clang-format on
```